database/seedMethods.js

```diff
@@ -2,14 +2,41 @@
 /* eslint-disable no-shadow */
 
 const Promise = require('bluebird');
-const { createData } = require('./sampleDataModel');
+const { createBook } = require('./sampleDataModel');
+const fs = require('fs');
+const fastCSV = require('fast-csv');
 
-// creates a data array for 100 BOOKS
 
-const createDataArray = () => {
+
+// function writeBooks() {
+//   while (i > 0) {
+//     if (!csvWriteStream.write(generateBooks())) {
```

Comment:
Looks good here. But you may need to alter your generateBooks() function later to pass in the index i to be used as an ID in your noSQL db later for querying a specific book ID.